### ISSUE TITLE
User wants support for device custom lat/long

### ISSUE THREAD

**Issue by thgreasi** [balena-io/open-balena-api#258]
The `custom latitude` & `custom longitude` device fields are currently not supported on openBalena.

**Comment by iKK001** [comment]
Any news on this issue ?  

i.e. custom lat/long is still not supported by the balena API , it seems...

```Swift
let balenaDevice = try await WebService.shared.fetch(from: self.balenaResource)
let location = balenaDevice.d[0].location
```
`location` still represents the non-custom (i.e. original) latitude/longitude, unfortunately....

**Comment by dfunckt** [comment]
Yeah, custom location support was added long after automatic reverse geocoding and for some reason it was implemented within balena-api, our internal codebase. It definitely belongs to open-balena-api though and should be easy to move the code over, just needs someone from the backend team to spend a few hours on it.
